demo/src/app/components/shared/api-docs/api-docs-class.component.html

```diff
@@ -29,20 +49,22 @@
 
   <ng-template [ngIf]="apiDocs.methods && apiDocs.methods.length">
     <section>
-      <h3>Methods</h3>
-      <table class="table table-sm table-hover">
+      <h4>Methods</h4>
+      <table class="table table-sm">
         <tbody>
-        <tr *ngFor="let method of apiDocs.methods">
-          <td class="label-cell"><code>{{method.name}}</code></td>
-          <td class="content-cell">
-            <div><i>Signature: </i><code>{{ methodSignature(method) }}</code></div>
-            <div><i>Return type: </i><code>{{ method.returnType }}</code></div>
-            <div style="margin: 10px 0">{{ method.description }}</div>
-          </td>
-        </tr>
+          <tr *ngFor="let method of apiDocs.methods">
+            <td class="label-cell"><code>{{method.name}}</code></td>
+            <td class="content-cell">
```

Comment:
I would have added 1 rem bottom padding (see attached screenshot) for readability

Reply:
done